Composer/packages/extensions/adaptive-form/src/components/fields/ExpressionField/ExpressionEditor.tsx

```diff
@@ -1,20 +1,24 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import React from 'react';
-import { FieldProps } from '@bfc/extension';
-import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { FieldProps, useShellApi } from '@bfc/extension';
+import { IntellisenseTextField } from '@bfc/intellisense';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
+import React from 'react';
+
+let intellisenseWsUrl = window.location.origin.replace(/^https/, 'wss').replace(/^http/, 'ws');
+intellisenseWsUrl = `${intellisenseWsUrl}/intellisense-language-server`;
 
 const ExpressionEditor: React.FC<FieldProps> = (props) => {
   const { id, value = '', onChange, disabled, placeholder, readonly, error } = props;
 
-  const handleChange = (e: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
-    onChange(newValue);
-  };
+  const { projectId } = useShellApi();
 
   return (
-    <TextField
+    <IntellisenseTextField
+      url={intellisenseWsUrl}
```

Comment:
Consider avoiding abbreviation in the name - intellisenseServerUrl

Reply:
Renamed to intellisenseWebSocketUrl